types/carbon-components-react/lib/components/Accordion/AccordionItem.d.ts

```diff
@@ -8,10 +8,12 @@ export interface HeadingClickData {
     isOpen: boolean;
 }
 
-export interface AccordionItemProps extends InheritedProps {
+export interface AccordionItemProps extends Omit<React.HTMLProps<InheritedProps>, "title"> {
```

Comment:
```suggestion
export interface AccordionItemProps extends Omit<InheritedProps, "title"> {
```
Any HTML props and common carbon props are already on the `InheritedProps` interface. There's no constraint on that type parameter of `HTMLProps`  but it is used as the type for the event handlers and ref so we're no longer extending from `InheritedProps`.

Reply:
Done! Appreciated, thanks!